### PR TITLE
Use relative path for attachments in JSON report

### PR DESCRIPTION
**Pull request by raoyitao**
* The "_attachments" directory is always near the main Testplan JSON
  report, so the reletive path should be used, thus the saved reports
  can be moved freely to any location, TPR script is able to display
  the report in web UI.

### CODE REVIEW

File: testplan/exporters/testing/base.py [hunk 1]
```diff
@@ -191,6 +192,6 @@ def save_attachments(report: TestReport, directory: str) -> Dict[str, str]:
             dst_path = os.path.join(directory, dst)
             makedirs(os.path.dirname(dst_path))
             copyfile(src=src, dst=dst_path)
-            moved_attachments[dst] = dst_path
+            moved_attachments[dst] = os.path.join(".", ATTACHMENTS, dst)
```
Comment (Pyifan) [inline testplan/exporters/testing/base.py:195]: if we need a relative path here, maybe change dst_path to relative path?

**Review by Pyifan — CHANGES_REQUESTED** [review]
This will need a news fragment as well

File: testplan/exporters/testing/base.py (hunk 1 above)
Comment (raoyitao) [reply testplan/exporters/testing/base.py:195]: I think not necessary. `os.path.join(".", ATTACHMENTS, dst)` is a string written into json report. If we actually copy a file, using the absolute path is much reliable that it does not matter where to execute script.

Comment (Pyifan) [reply testplan/exporters/testing/base.py:195]: But move_attachements[dst] shall be the actual place where we copy attachment to.

**Review by Pyifan — CHANGES_REQUESTED** [review]

**Pull request merged**